internal/api/auth/auth.go

```diff
@@ -0,0 +1,189 @@
+package auth
+
+import (
+	"errors"
+	"fmt"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/brocaar/lorawan"
+	jwt "github.com/dgrijalva/jwt-go"
+)
+
+// Claims defines the struct containing the token claims.
+type Claims struct {
```

Comment:
Here you go: https://github.com/brocaar/loraserver/pull/53/commits/381711e58a7194efcfeb5b0b6ba5174943a4d147